classes/class-wc-rest-connect-shipping-label-controller.php

```diff
@@ -72,10 +83,20 @@ public function update_items( $request ) {
 				);
 			}
 			$labels_data[] = $label_data->label;
+			$labels_order_meta[] = array(
+				'label_id' => $label_data->label->label_id,
+				'provider_label_id' => $label_data->label->provider_label_id,
+				'refundable_amount' => $label_data->label->refundable_amount,
+				'created' => $label_data->label->created,
+				'carrier_id' => $settings[ 'carrier' ],
+				'service_name' => $settings[ 'carrier' ], // TODO: Get the used service name from the request
```

Comment:
I will implement this after the `Rates Step` PR is merged, I will need to touch code from there. The only real consequence is that the `Ship via` item in the meta-box will simply display "usps" instead of "USPS Priority blah blah".